src/fastoad/_utils/strings.py

```diff
@@ -60,20 +59,28 @@ def get_float_list_from_string(text: str):
     # Deals with multiple values in same element. numpy.fromstring can parse a string,
     # but we have to test with either ' ' or ',' as separator. The longest result should be
     # the good one.
-    try:
-        value1 = np.fromstring(text_value, dtype=float, sep=" ").tolist()
-    except ValueError:
-        pass
+    with warnings.catch_warnings():
+        # np.fromstring issues these messages for faulty strings
+        #   DeprecationWarning: string or file could not be read to its end due to unmatched data;
+        #   this will raise a ValueError in the future.
+        # The processing of ValueError is ready, so let's ensure we get through the
+        # error management
+        warnings.filterwarnings("error", category=DeprecationWarning)
 
-    try:
-        value2 = np.fromstring(text_value, dtype=float, sep=",").tolist()
-    except ValueError:
-        pass
+        try:
+            value1 = np.fromstring(text_value, dtype=float, sep=" ").tolist()
+        except (ValueError, DeprecationWarning):
+            value1 = []
```

Comment:
That's one way of suppressing warnings 😅

Reply:
There is more to it. The lines above (context manager, and warning filter) are important.